roles/pulp_webserver/vars/Fedora.yml

```diff
@@ -4,6 +4,8 @@
 pulp_seboolean_packages:
   - python3-libselinux
   - python3-libsemanage
+pulp_firewall_dependencies:
```

Comment:
Create a vars file for debian with "firewalld".

Or better yet, for debian with "python3-firewall" and for debian-10 with "firewalld".

Reply:
better debian-10 without `pulp_firewall_dependencies` as not needed (everything needed in deb package firewalld)

Reply:
Oh, right, we install firewald in the earlier task.